packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -324,13 +324,25 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
         return;
       }
 
-      const localStream = callState.localVideoStreams.find((item) => item.mediaStreamType === 'Video');
-      if (!localStream) {
-        return;
+      // Find the first available stream, if there is none, then get the first stream
+      const localVideoStream = callState.localVideoStreams.find((item) => item.mediaStreamType === 'Video');
+      const screenShareStream = callState.localVideoStreams.find((item) => item.mediaStreamType === 'ScreenSharing');
```

Comment:
TODO: update this for onDisposeLocalStream also. It will need a new arg to distinguish which stream should be disposed off (we do this in a very hacky way for remote streams so we should update that also in the process)

Reply:
Updated to have disposeLocalScreenShareStream